logic_functions.py

```diff
@@ -21,9 +21,9 @@ def get_fresh_board():
 
 
 
-def get_next_board(squares, i, x_is_next):
-    if squares[i] == "X":
-        x_is_next = "O"
+def get_next_board(squaresB, i, x_is_next):
```

Comment:
Why did you change the squares to squaresB here?

Reply:
in the Board.js file it says getNextBpard takes squaresB.

Reply:
Does the name of a variable have anything to do with the name of the parameter in a function signature?

Reply:
ugh no lol my bad